docs/index.md

```diff
@@ -1,17 +1,18 @@
 ## Charmed Kafka Documentation
```

Comment:
To keep a consistent home page structure across charms, order would be:
1. Title
2. Description
3. "In this documentation"
4. "Project and community"
5. License